lib/driver.js

```diff
@@ -332,6 +324,14 @@ class IosDriver extends BaseDriver {
       return;
     }
 
+      // if we have an ipa file, set it in opts
+      if (this.opts.app) {
```

Comment:
Looking into this a little deeper, is there a reason there is a seperate `installIpa` method? I could be missing something, but it lloks like .ipa's are always unziped in the BaseDriver: 
https://github.com/appium/appium-base-driver/blob/b1323602d46fc7967913758e3b2a913d1b0f9bc8/lib/helpers.js#L22